### PR TITLE
normalize indentation and json styling

### PR DESCRIPTION
**Pull request by sgoings**
My OCD took over while reading through this documentation. 🤣 Hopefully this helps others come up to speed quicker on ARM!

### CODE REVIEW

File: v1.0/proxy-api-reference.md [hunk 1]
```diff
@@ -37,7 +37,7 @@ Examples of action names include: "restartVM" or "listStorageKeys".
 **Request Body**
 
     {
-    "resourceDefinedProperty": "Resource defined value";
+      "resourceDefinedProperty": "Resource defined value";
```
Comment (ultimateboy) [inline v1.0/proxy-api-reference.md]: what's up with the `;` at the end here?

File: v1.0/common-api-details.md [hunk 2]
```diff
@@ -89,23 +89,23 @@ The resource providers must return the \*code\* and \*message\* fields; however,
...
+        "code": "BadArgument",
+        "message": "The provided database &#39;foo&#39; has an invalid username.",
+        "target": "query",
+        "details": [
+          {
+            "code": "301",
+            "target": "$search"
+            "message": "$search query option not supported",
+          }
+        ]
```
Comment (ultimateboy) [inline v1.0/common-api-details.md]: There is a missing comma here.

File: v1.0/resource-api-reference.md [hunk 3]
```diff
@@ -50,23 +50,23 @@ The resource group name and resource name **MUST** come from the URL and not the
...
+        "name": "sku code, such as P3",
+        "capacity": {number}
+      },
+      "plan": {
+        "name": "User defined name of the 3rd Party Artifact",
+        "publisher": "Publisher of the 3rd Party Artifact ",
+        "product": "OfferID for the 3rd Party Artifact ",
+        "promotionCode": "Promotion Code",
+        "version": "Version of the 3rd Party Artifact"
+      }
```
Comment (ultimateboy) [inline v1.0/resource-api-reference.md]: Missing comma here

File: v1.0/resource-api-reference.md [hunk 4]
```diff
@@ -349,23 +351,27 @@ The paging approach required by ARM is server side paging, as described below.
...
-        "location": "North US",
-        "properties": { "comment: "Resource defined structure" }.
-        "kind" : "resource kind"
-    }
-    ],
-    "nextLink": "{originalRequestUrl}?$skipToken={opaqueString}"
+          "id": "{url to resource 1}",
+          "name": "Name1",
+          "type": "{resourceProviderNamespace}/{resourceType}",
+          "location": "North US"
```
Comment (ultimateboy) [inline v1.0/resource-api-reference.md]: Missing comma here

File: v1.0/subscription-lifecycle-api-reference.md [hunk 5]
```diff
@@ -25,17 +25,21 @@ This API uses the &quot;system&quot; version of 2.0 because it can be triggered
...
-        "properties": {
-
-                "tenantId":"ac430efe-1866-4124-9ed9-ee67f9cb75db",
-                "locationPlacementId":"Internal_2014-09-01",
-                "quotaId":"Default_2014-09-01",
-                "registeredFeatures": [{ "name": "<featureName>", "state": "Registered" }]
-       }
+      "state": "Registered" | "Unregistered" | "Warned" | "Suspended" | "Deleted",
+
+      "registrationDate": "Tue, 15 Nov 1994 08:12:31 GMT"
```
Comment (ultimateboy) [inline v1.0/subscription-lifecycle-api-reference.md]: Missing comma here

**Comment by ravbhatnagar** [comment]
@sgoings - Looks like there are some merge conflicts. Can you please resolve?

**Pull request merged**